Serato-Now-Playing/serato_now_playing.py

```diff
@@ -513,28 +435,30 @@ def on_filebutton_clicked(self):
             startdir = os.path.dirname(startfile)
         else:
             startdir = '.'
-        filename = QFileDialog.getOpenFileName(self.window, 'Open file',
+        filename = QFileDialog.getSaveFileName(self.window, 'Open file',
```

Comment:
This change absolutely requires a documentation change since the file no longer has to be pre-created as per docs.